website/content/docs/contribution.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Contribution"
+description: "Contributing Guidelines"
+
+weight: 5
+---
+
+First off, thanks for taking the time to contribute!
```

Comment:
I think we shouldn't write this. They haven't contributed yet, rather are deciding to contribute. So something like `First off, thank you for showing your interest in contributing the project`